modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/containers/SideMenu/index.js

```diff
@@ -47,33 +47,43 @@ const SideMenu = () => {
 		isOpenedProductsMenu,
 	]);
 
+	const isLiferayExperienceCloud = subscriptionGroups?.some(
```

Comment:
```suggestion
	const hasLiferayExperienceCloud = subscriptionGroups?.some(
```

Reply:
Also, it should be good use a useMemo here to avoid do this loop

Reply:
Nice, Jair! Thank you!